tensorboard/components/experimental/plugin_lib/test/test.ts

```diff
@@ -89,4 +89,16 @@ describe('plugin lib integration', () => {
       });
     });
   });
+
+  describe('lib.core', () => {
+    describe('#getURLPluginData', () => {
+      it('returns URL data', async function() {
+        tf_globals.setFakeHash('sample_plugin&p.sample_plugin.foo=bar');
```

Comment:
How does this test work? Is this setting the hash on the parent frame and are we testing both lib and the host?

Also, please add more complex cases :) Ideas:
- weird values
- p.wrong_plugin.foo
- p.sample_plugin.foo.bar